stock_picking_validate_jobcan_wf/models/stock_picking.py

```diff
@@ -25,9 +25,22 @@ def _is_outgoing(self):
             "transit",
         ] and self.location_dest_id.usage not in ["internal", "transit"]
 
+    def _receipt_return_picking(self):
+        self.ensure_one()
+        if not self.move_ids:
+            return False
+        for move in self.move_ids:
+            if not (move.origin_returned_move_id and move._is_out()):
+                return False
+        return True
```

Comment:
Non-blocking. I guess it doesn't make much difference.
```suggestion
        if self.move_ids and self.move_ids == self.move_ids.filtered(
            lambda x: x._is_out() and x.origin_returned_move_id
        ):
            return True
        return False
```